Modules/Filtering/Thresholding/include/itkIntermodesThresholdImageFilter.h

```diff
@@ -93,31 +93,15 @@ class IntermodesThresholdImageFilter : public HistogramThresholdImageFilter<TInp
   using HistogramType = typename Superclass::HistogramType;
   using CalculatorType = IntermodesThresholdCalculator<HistogramType, InputPixelType>;
 
-  void
-  SetMaximumSmoothingIterations(SizeValueType maxSmoothingIterations)
-  {
-    m_IntermodesCalculator->SetMaximumSmoothingIterations(maxSmoothingIterations);
-  }
 
-  SizeValueType
-  GetMaximumSmoothingIterations()
-  {
-    return (m_IntermodesCalculator->GetMaximumSmoothingIterations());
-  }
+  itkSetMacro(UseInterMode, SizeValueType);
```

Comment:
Should this be `MaximumSmoothingIterations`? `UseInterMode` is declared few lines later.

Reply:
Good catch!